appengine/standard/endpoints-frameworks-v2/backend/README.md

```diff
@@ -0,0 +1,38 @@
+## Endpoints Frameworks v2 Python Sample
+
+This demonstrates how to use Google Cloud Endpoints Frameworks v2 on Google App Engine Standard Environment using Python.
+
+This sample consists of two parts:
+
+1. The backend
+2. The clients
+
+## Running Locally
+
+For more info on running Standard applications locally, see [the getting started documentation](https://cloud.google.com/appengine/docs/python/quickstart).
+
+Install the Endpoints Frameworks v2 library into a vendor directory:
+
+    $ pip install -t lib TODO
```

Comment:
Not necessarily, I'm checking with folks to find out what the name will be.
I'm fine with leaving this pull up for a bit, just wanted to start the process with what we've got.